### PR TITLE
Check airlock review logic in e2e tests

### LINKED ISSUE #2734: Check Airlock Review logic in e2e tests
In e2e tests, do the following steps:
- Deploy Airlock Import Review workspace alongside base workspace 
- For base workspace, set up Airlock review configuration
- For an import request, create a review VM and check that it got deployed
- After the request was approved, check that the VM got cleaned up

### PR DESCRIPTION
**Pull request by tanya-borisova**
# Resolves #2734 

## What is being addressed

Add a new test creating an Airlock request, creating a Review VM for it and checking that the VM will be deleted after the request is approved. 

## How is this addressed

- Created additional fixtures for creating workspace service (for a "normal" workspace), and creating workspace and workspace service for import review workspace
- Add a test under `airlock` selector that checks the logic for review VM

### CODE REVIEW

File: e2e_tests/test_shared_services.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import logging
 
 from resources.resource import disable_and_delete_resource, post_resource
-from helpers import get_shared_service_id_by_name
+from helpers import get_shared_service_by_name
```
Comment (guybartal) [inline e2e_tests/test_shared_services.py:5]: make sure all calls to this renamed function was changed,
line number 100 still uses `get_shared_service_id_by_name`

**Review by guybartal — APPROVED** [review]
LGTM,
left one comment for missing function renaming (also falls during linting).
plus need to make sure there are no time outs during E2E tests run on Github, we might need to adjust the main TO accordingly.

in the connected issue #2734 there's tasks related to export flow tests which are not implemented, consider creating a linked issue.

File: e2e_tests/test_shared_services.py (hunk 1 above)
Comment (tanya-borisova) [reply e2e_tests/test_shared_services.py:5]: Good catch, thank you!

**Comment by tanya-borisova** [comment]
/test-force-approve

A successful run on branch (with airlock selector): https://github.com/microsoft/AzureTRE/actions/runs/3647396050/jobs/6163401606

**Pull request merged**